src/main/java/com/stripe/model/Plan.java

```diff
@@ -22,6 +24,10 @@
   Map<String, String> metadata;
   String nickname;
   ExpandableField<Product> product;
+  List<PlanTier> tiers;
+  PlanTransformUsage transformUsage;
+  String tiersMode;
```

Comment:
`tiersMode` should be before `transformUsage`.

Reply:
Would be cool if a linter could catch this :-)